timepiece/utils/views.py

```diff
@@ -10,6 +10,12 @@ def class_decorator(View):
     return class_decorator
 
 
+def format_totals(entry_dict, key="sum"):
```

Comment:
We should have a test that explicitly tests this utility in `timepiece.tests.test_utils`